collect_app/src/main/java/org/odk/collect/android/preferences/screens/ServerPreferencesFragment.java

```diff
@@ -140,6 +140,12 @@ public void addAggregatePreferences() {
         serverUrlPreference.setOnPreferenceChangeListener(createChangeListener());
         serverUrlPreference.setSummary(serverUrlPreference.getText());
 
+        serverUrlPreference.setOnBindEditTextListener(editText -> {
+            editText.selectAll();
+            editText.setCompoundDrawablesWithIntrinsicBounds(0, 0, R.drawable.ic_arrow_drop_down, 0);
```

Comment:
It looks like this line and the line below are no longer needed. Sorry, I think they got removed while you were working on this!

Reply:
@virtualpathum these lines still need to be removed and then I can look at adding a test for you!